resources/assets/less/wiki.less

```diff
@@ -16,23 +17,178 @@
 *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 *
 */
-@import "app";
 
-.wiki-header {
-  .at2x-simple("/images/headers/tournament.jpg");
-  background-size: cover;
+.language-labels {
+    a {
+        padding-left: 5%;
+    }
+}
 
-  .text {
-    padding-left: 130px;
+.wiki-contents {
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
 
-    h1, h2 {
-      .default-text-shadow;
-      color: #fff;
+.wiki-nav-container {
+    right: 30px;
+    .wiki-button {
+        span {
+            font-size: 170%;
+        }
+    }
+    .top {
+        margin-top: -14%;
+        .wiki-inner {
+            font-size: 60%;
+            color: #FFF;
+        }
+        .wiki-outer {
+            color: #0088AB;
+        }
+    }
+}
 
-      font-style: normal;
+.wiki-nav-bottom {
+    //container
+    &:extend(.row);
+    z-index: 1000;
+    position: fixed;
+    width: 15%;
+    max-width: 160px;
+    text-align: center;
+    bottom: 25%;
+    .bottom {
+        .wiki-inner {
+            font-size: 60%;
+            color: #0088AB;
+        }
+        .wiki-outer {
+            color: #FFF;
+            text-shadow: 4px 4px 4px #A8A8A8;
+        }
     }
+    a {
+        display: block;
+        margin-bottom: 3px;
+        text-align: center;
+    }
+}
 
-    h1 { font-size: 32px; margin-bottom: 10px; }
-    h2 { font-size: 16px; }
-  }
+.wiki-nav {
+    &:extend(.row);
+    z-index: 1000;
+    position: fixed;
+    box-shadow: 0px 1px 5px rgba(0, 0, 0, 0.4);
+    -moz-border-radius: 6px;
+    border-radius: 6px;
+    width: 15%;
+    max-width: 160px;
+    height: 35%;
+    min-height: 380px;
+    text-align: center;
+    background-color: #FFFFFF;
+    a {
+        display: block;
+        margin-bottom: 3px;
+        text-align: center;
+    }
+}
+
+#hideWikiNav:hover {
+    opacity: 0.75;
+}
+
+#showWikiNav:hover {
+    opacity: 0.75;
 }
+
+.wiki-page {
```

Comment:
this top class isn't even needed. Just use the page-compact modifier of osu-layout__row.